packages/altive_lints/lib/src/lints/avoid_single_child.dart

```diff
@@ -85,4 +96,12 @@ class AvoidSingleChild extends DartLintRule {
       }
     });
   }
+
+  bool _hasSingleChild(CollectionElement element) {
+    if (element is SpreadElement && element.expression is ListLiteral) {
+      final spreadElement = element.expression as ListLiteral;
+      return spreadElement.elements.length != 1;
+    }
```

Comment:
![badge](https://img.shields.io/badge/review-ask-informational)

What does the name `_hasSingleChild` mean when `length != 1`, what does the name `_hasSingleChild` mean?It seems intuitive if `length == 1`...

Reply:
Indeed, at a quick glance, it seems that the function name does not match the actual situation.

The function name _hasSingleChild seems to return true in the case of a single element, so the function name should be changed to _hasMultipleChild to make it clear that it returns true when there are multiple child elements.

Reply:
Names are important. Make sure you name it with proper thought and self-review to make sure it is correct.